cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/config/ConcurrencyConfigTest.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.config;
+
+import com.linkedin.kafka.cruisecontrol.config.constants.ExecutorConfig;
+import org.apache.kafka.common.config.ConfigException;
+import org.easymock.EasyMock;
+import org.junit.Test;
+
+import static org.junit.Assert.assertThrows;
+
+
+public class ConcurrencyConfigTest {
```

Comment:
2 nits:

1. Consider using `import com.linkedin.kafka.cruisecontrol.config.constants.ExecutorConfig.*; `
2. Consider creating a `private static final String GET_INT_METHOD = "getInt"`

Reply:
1. I typically refrain from using wildcard imports unless all static variables are used in the importing class.
2. Makes sense!